aclk/aclk_query.c

```diff
@@ -112,7 +112,7 @@ static int http_api_v2(struct aclk_query_thread *query_thr, aclk_query_t query)
     now_realtime_timeval(&w->tv_ready);
 
     if (query->timeout) {
-        double in_queue = (int)dt_usec(&w->tv_in, &w->tv_ready) / 1000;
+        double in_queue = ((int) dt_usec(&w->tv_in, &w->tv_ready)) / 1000.0;
```

Comment:
This will work for sure, but why did you not cast directly `dt_usec` for `double`? I agree that on `X86_64` results will always go to `rax` and after this the result will be moved to a proper float point register, my suggestion is to explicit that everything will be converted to double to execute the math.

Reply:
Switching to int as this will avoid another conversion in the next line

Reply:
Perfect,  will test it right now.